sky/tools/objcopy.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import os
+import subprocess
+import sys
+
+def main():
+  parser = argparse.ArgumentParser(description='Convert a data file to an object file')
+  parser.add_argument('--objcopy', type=str, required=True)
+  parser.add_argument('--input', type=str, required=True)
+  parser.add_argument('--output', type=str, required=True)
+
+  args = parser.parse_args()
+
+  input_dir, input_file = os.path.split(args.input)
+  output_path = os.path.abspath(args.output)
+
+  subprocess.check_call([
+    args.objcopy,
+    '-I', 'binary',
+    '-O', 'elf32-littlearm',
```

Comment:
This isn't architecture agnostic. Thinking of release modes for Android `aarch64`, `x64`, etc..

Reply:
done